util/test/demos/main.cpp

```diff
@@ -139,7 +139,7 @@ void NuklearShutdown()
   UnregisterClassW(wc.lpszClassName, wc.hInstance);
 }
 
-#else
+#elif defined(__LINUX__)
```

Comment:
I think the standard macro for linux is `__linux__` lowercase.

Reply:
Thanks not sure why I chose __LINUX__ instead of __linux__